ml/ml.cc

```diff
@@ -146,8 +190,128 @@ int test_ml(int argc, char *argv[]) {
     (void) argc;
     (void) argv;
 
+    if(test_ml_anomaly_info_api_sql() == 0) {
+        fprintf(stderr, "PASSED testing ML API SQL query; SQL_SELECT_ANOMALY_RATE_INFO\n");
+    }
+
```

Comment:
We should use the `googletest` framework for these tests.

Reply:
Are you addressing CI operations. Otherwise, I would leave as is, as it is doing the job more straight forward.